sdk/python/feast/data_source.py

```diff
@@ -496,6 +495,15 @@ def source_datatype_to_feast_value_type() -> Callable[[str], ValueType]:
         raise NotImplementedError
 
 
+class RequestDataSource(RequestSource):
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "The 'RequestDataSource' class is deprecated and was renamed to RequestSource. Please use RequestSource instead.",
```

Comment:
Also add that this class will be removed Feast 0.23

Reply:
Fixed!